### ISSUE TITLE
Look at the regional options in sparta

### ISSUE THREAD

**Issue by drnickisaac** [drnickisaac/sMon-Amphibia#3]
This should be a good way for you to explore variation among federal states, and even partially control for spatial effects

**Comment by EichenbergBEF** [comment]
This looks quite promising. I wonder it the 'regional_aggs' argument is what were reffering to when you said it only allows for three different levels here. 
At the moement this would be fine, because if I remember right, the data that we analyzed in the ws for amphibains is from 3 federal states, only. Same with orthoptera, and for odonata I think it is only 2 states.

**Comment by UlliSchulte** [comment]
In the workshop we analyzed data of 7 states (SH, BE, ST, TH, HE, RP and NW) with Julian date, but the plan is to analyze the data of 3 states where you got the original data directly from the data owners (NW, SN, ST).

**Comment by EichenbergBEF** [comment]
Okay, this leaves some work to still be accomplished. We need to think of how to integrate julian date and non-julian date data in one model. If I got that right it is for the state of saxony (SN) that we do not have julian dates, right? For NW and ST we obviously have these data.

**Comment by UlliSchulte** [comment]
Right. But SN should habe Julian dates. Please ask Holger Lueg. If SN has no Julian dates data, than SH would be an alternative (Arne Drews is very cooperative).

**Comment by EichenbergBEF** [comment]
Okay. I have updated the original data you, Ulrich, gave us with the data provided by Saxony (Holger Lueg). There we have date precision information now. The datasets are also on the portal (see mail later today) so that people wanting to work with them can all use the same datasets. I also attached a map for the geographical distribution of occurrence data of the different nested levels we have.

@drnickisaac: I also ran the models during the weekend (some of them are still pending). The models specifically take the three states NRW, ST and SN into account and also work with the gegional codes. I will commit the script, soon. By now, I got the analyses for all newts and 4 frog spec up to now. I will also put them on the portal, with restricuted acces for Ulrich, Nick, Diana and me. More results to come :-)
![geographical_distribution_occurrence_data](https://user-images.githubusercontent.com/10739274/35508002-0a090e24-04ef-11e8-847c-0ff56dc2a155.jpg)
